tests/testthat/test-translate-window.R

```diff
@@ -49,3 +49,19 @@ test_that("can override frame of recycled functions", {
   )
 
 })
+
+test_that("win_rank() is accepted by the sql_translator", {
```

Comment:
This just testing the print method?

Reply:
I think I started with `expect_silent()` but felt like we could do more.  Figured that by testing the print, we could catch unintended changes to that output.

Reply:
I think this test should live in `test-translate-sql-helpers.r`.